samples/dashboards/demo/live-weather/demo.js

```diff
@@ -413,14 +413,12 @@ async function setupDashboard() {
                             board, activeCity,
                             'temperature', true, false
                         );
-                    },
-                    afterLoad: function () {
-                        this.cell.setActiveState();
                     }
                 },
                 states: {
                     active: {
-                        enabled: true
+                        enabled: true,
+                        activated: true
```

Comment:
I would replace it with one of them: "initialized", "visible", "isActive", "isActiveState"

`active.activated` does not sound intuitive to me.